hendrikjoosten/pairmeup
# Page navigation

### What 🕵️ 🔍
This adds page navigation, a bolt font and moves the folder structure up one level. 
__________________________________________________________________________________________
### How to test 🥼 🔬
Run npm run dev and see if any errors pop up 
__________________________________________________________________________________________
## References 📝 🔗
[How to add fonts ](https://www.geeksforgeeks.org/how-to-add-custom-local-fonts-in-next-js/)
